frontend/src/assets/close.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="29.25" height="29.25" viewBox="0 0 29.25 29.25">
+  <g id="Icon_ionic-ios-close-circle-outline" data-name="Icon ionic-ios-close-circle-outline" transform="translate(-3.375 -3.375)">
+    <path id="Pfad_15" data-name="Pfad 15" d="M23.295,21.705,19.589,18l3.705-3.705a1.124,1.124,0,0,0-1.589-1.589L18,16.411l-3.705-3.705a1.124,1.124,0,0,0-1.589,1.589L16.411,18l-3.705,3.705a1.086,1.086,0,0,0,0,1.589,1.116,1.116,0,0,0,1.589,0L18,19.589l3.705,3.705a1.129,1.129,0,0,0,1.589,0A1.116,1.116,0,0,0,23.295,21.705Z" fill="#7b7b7b"/>
+    <path id="Pfad_16" data-name="Pfad 16" d="M18,5.344A12.651,12.651,0,1,1,9.049,9.049,12.573,12.573,0,0,1,18,5.344m0-1.969A14.625,14.625,0,1,0,32.625,18,14.623,14.623,0,0,0,18,3.375Z" fill="#7b7b7b"/>
+  </g>
+</svg>
```

Comment:
There are some unused properties created by design tools. 
You can safely remove `id` and `data-name` to reduce the file-size.
https://www.svgminify.com/ or https://imageoptim.com/ could be helpful here.

This issue happens multiple times.

Reply:
Thank you for the links!